aws-ecs/artifacts/cloudformation-template.yaml

```diff
@@ -36,193 +41,198 @@ Resources:
-            - ec2:DescribeLaunchTemplates
-            - ec2:DescribeNatGateways
-            - ec2:DescribeNetworkAcls
-            - ec2:DescribeRouteTables
-            - ec2:DescribeSecurityGroups
-            - ec2:DescribeSubnets
-            - ec2:DescribeTags
-            - ec2:DescribeVpcAttribute
-            - ec2:DescribeVpcClassicLink
-            - ec2:DescribeVpcClassicLinkDnsSupport
-            - ec2:DescribeVpcs
-            - ec2:DescribeAvailabilityZones
-            - ec2:ModifySubnetAttribute
-            - ec2:ModifyVpcAttribute
-            - ec2:ModifyLaunchTemplate
-            - ec2:RevokeSecurityGroupEgress
-            - ec2:RunInstances
-            - ec2:DeleteNetworkAclEntry
-            - ec2:CreateNetworkAclEntry
-            - ecs:CreateCluster
-            - ecs:ListTagsForResource
-            - ecs:TagResource
-            - ecs:DescribeClusters
-            - ecs:DescribeCapacityProviders
-            - ecs:CreateCapacityProvider
-            - ecs:PutClusterCapacityProviders
-            - ecr:CreateRepository
-            - ecr:DescribeRepositories
-            - ecr:ListTagsForResource
-            - ecr:TagResource
-            - iam:AttachRolePolicy
-            - iam:CreateOpenIDConnectProvider
-            - iam:UpdateAssumeRolePolicy
-            - iam:TagOpenIDConnectProvider
-            - iam:CreatePolicy
-            - iam:CreatePolicyVersion
-            - iam:CreateRole
-            - iam:CreateServiceLinkedRole
-            - iam:GetOpenIDConnectProvider
-            - iam:GetPolicy
-            - iam:GetPolicyVersion
-            - iam:GetRole
-            - iam:GetRolePolicy
-            - iam:ListAttachedRolePolicies
-            - iam:ListRolePolicies
-            - iam:PassRole
-            - iam:PutRolePolicy
-            - iam:TagPolicy
-            - iam:TagRole
-            - kms:CreateAlias
-            - kms:CreateGrant
-            - kms:CreateKey
-            - kms:DescribeKey
-            - kms:GetKeyPolicy
-            - kms:GetKeyRotationStatus
-            - kms:ListAliases
-            - kms:ListResourceTags
-            - kms:PutKeyPolicy
-            - kms:TagResource
-            - logs:CreateLogGroup
-            - logs:TagLogGroup
-            - logs:DescribeLogGroups
-            - logs:ListTagsLogGroup
-            - logs:PutRetentionPolicy
-            - logs:TagResource
-            - route53:ChangeTagsForResource
-            - route53:CreateHostedZone
-            - route53:GetChange
-            - route53:GetHostedZone
-            - route53:ListResourceRecordSets
-            - route53:ListTagsForResource
-            - route53:ChangeResourceRecordSets
-            - s3:GetObject
-            - s3:ListBucket
-            - s3:PutObject
+              - ec2:AllocateAddress
+              - ec2:AssociateRouteTable
+              - ec2:AttachInternetGateway
+              - ec2:AuthorizeSecurityGroupEgress
+              - ec2:DescribeAvailabilityZones
```

Comment:
I think it'll work regardless, but it looks like `ec2:DescribeAvailabilityZones` is in the list 3 times.

Reply:
ahh, @jordan-acosta - I guess it was there from originally testing ECS.